### PR TITLE
Asset mapper

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
Add asset mapper to our framework and remove Webpack/node and such. Works together with changes to our [sumocoders/application-skeleton](https://github.com/sumocoders/application-skeleton/pull/135)

### CODE REVIEW

File: assets/fonts/lato-black-webfont.ttf
Comment (tijsverkoyen) [inline assets/fonts/lato-black-webfont.ttf:1]: Waarom moeten deze niet in `assets-public`?

File: assets/images/user-background.jpg
Comment (tijsverkoyen) [inline assets/images/user-background.jpg:1]: Wat is deze file?

Comment (jonasdekeukelaere) [reply assets/images/user-background.jpg:1]: Alle images verwijderd uit core bundle 83d250c

File: docs/frontend/asset-mapper.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+Our main CSS is currently located in the framework-core-bundle as this still needs to be compile with scss for Bootstrap.
+This CSS is included in the `assets/styles/style.scss` file.
+
+To compile the CSS with SASS, you can run the following command:
+
+```bash
+symfony console sass:build
+```
+
```
Comment (tijsverkoyen) [inline docs/frontend/asset-mapper.md:55]: Onderstaande nog toevoegen:

If you are working locally and doing changes, and as long as we are using SASS you can run:

```bash
symfony console sass:build --watch
```

Comment (jonasdekeukelaere) [reply docs/frontend/asset-mapper.md:55]: Fixed d295100

**Pull request merged**